tests/functional/test_model_completeness.py

```diff
@@ -34,15 +39,48 @@ def test_paginators_and_waiters_are_not_lost_in_new_version(
 ):
     # Make sure if a paginator and/or waiter exists in previous version,
     # there will be a successor existing in latest version.
-    loader = Loader()
     try:
-        loader.load_service_model(service_name, type_name, previous_version)
+        LOADER.load_service_model(service_name, type_name, previous_version)
     except DataNotFoundError:
         pass
     else:
         try:
-            loader.load_service_model(service_name, type_name, latest_version)
+            LOADER.load_service_model(service_name, type_name, latest_version)
         except DataNotFoundError as e:
             raise AssertionError(
                 f"{type_name} must exist for {service_name}: {e}"
             )
+
+
+def _endpoint_rule_set_cases():
+    for service_name in AVAILABLE_SERVICES:
+        versions = LOADER.list_api_versions(service_name, 'service-2')
+        for version in versions:
+            yield service_name, version
+
+
+@pytest.mark.parametrize(
+    "service_name, version",
+    _endpoint_rule_set_cases(),
+)
+def test_all_endpoint_rule_sets_exist(service_name, version):
+    """Tests the existence of endpoint-rule-set-1.json for each service
+    and verifies that content is present."""
+    type_name = 'endpoint-rule-set-1'
+    data = LOADER.load_service_model(service_name, type_name, version)
+    assert len(data['rules']) >= 1
+
+@pytest.mark.parametrize("service_name", AVAILABLE_SERVICES)
+def test_all_endpoint_tests_exist(service_name):
+    """Tests the existence of endpoint-tests-1.json for each service
+    and verifies that content is present."""
+    endpoint_tests_file = os.path.join(TEST_DATA_DIR, service_name, 'endpoint-tests-1.json')
+    with open(endpoint_tests_file) as f:
+        data = json.load(f)
+    assert len(data['testCases']) >= 1
```

Comment:
This is now also covered in my branch (currently in WIP PR https://github.com/jonemo/botocore/pull/1) together with all other tests that need to access the `tests/functional/endpoint-rules` folder.